packages/cardano-graphql/src/sdk.ts

```diff
@@ -1007,6 +1199,7 @@ export type MultiPolygonRef = {
 
 export type Mutation = {
   __typename?: 'Mutation';
+  addBlock?: Maybe<AddBlockPayload>;
```

Comment:
Can we keep the scoped provider types separate, then merge to compose to support multiple versions that include different surfaces?

Reply:
I think so, but it's something that can be addressed separately, so let's keep it out of scope of this PR.